bindings/c/include/opendal.h

```diff
@@ -192,17 +199,37 @@ extern "C" {
 #endif // __cplusplus
 
 /*
- Returns a result type [`opendal_result_op`], with operator_ptr. If the construction succeeds
- the error is nullptr, otherwise it contains the error information.
+ Uses an array of key-value pairs to initialize the operator based on provided scheme.
+
+ # Example
+
+ Following is a C example.
```

Comment:
I think we don't need this in a `.h` file?

Reply:
The comment in the header is auto generated from the Rust code.
I will check cbindgen doc to see if there is a way of avoiding this.

Reply:
Oh, that's fine. Let's keep AS-IS.